scripts/etc/init.d/rcS

```diff
@@ -834,6 +835,12 @@ touch /rootfs/etc/network/interfaces || fail
     fi
 } >> /rootfs/etc/network/interfaces || fail
 
+if [ "${disable_predictable_nin}" = "1" ]; then
+    # as described here: https://www.freedesktop.org/wiki/Software/systemd/PredictableNetworkInterfaceNames
+    # adding net.ifnames=0 to /boot/cmdline
+    cmdline="${cmdline} net.ifnames=0"
```

Comment:
hm... could you double check `net.ifnames=0` is actually added to `cmdline.txt`?